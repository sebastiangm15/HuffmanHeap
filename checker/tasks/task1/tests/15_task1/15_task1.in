15
7 A1
5 A2
3 A3
9 A4
8 A5
4 A6
12 A7
10 A8
6 A9
15 A10
13 A11
11 A12
2 A13
14 A14
1 A15
