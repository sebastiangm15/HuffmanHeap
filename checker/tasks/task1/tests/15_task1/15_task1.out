120-A15A13A3A9A7A11A14A10A1A5A4A6A2A8A12 
51-A15A13A3A9A7A11A14 69-A10A1A5A4A6A2A8A12 
24-A15A13A3A9A7 27-A11A14 30-A10A1A5 39-A4A6A2A8A12 
12-A15A13A3A9 12-A7 13-A11 14-A14 15-A10 15-A1A5 18-A4A6A2 21-A8A12 
6-A15A13A3 6-A9 7-A1 8-A5 9-A4 9-A6A2 10-A8 11-A12 
3-A15A13 3-A3 4-A6 5-A2 
1-A15 2-A13 
